packages/sdk/src/evm/common/gas-price.ts

```diff
@@ -188,8 +188,8 @@ export async function getPolygonGasPriorityFee(
       const fixedFee = parseFloat(priorityFee).toFixed(9);
       return utils.parseUnits(fixedFee, "gwei");
     }
-  } catch (e) {
-    console.error("failed to fetch gas", e);
+  } catch {
+    // if the gas station is down, return the default gas fee
```

Comment:
Worried about this error. If they changed the response maybe we just need to update it ?

Reply:
Most likely 403s (at least those happen in Unity sometimes) but yeah would need to debug 🤔

Reply:
seems it was just down, works fine now, I think this is the right behavior if that's the case